rpi/Date.pm

```diff
@@ -0,0 +1,50 @@
+package Date;
+
+use strict;
+use warnings;
+
+use Time::Piece;
+use Carp qw( croak );
+
+sub new {
+  bless {}, shift;
+}
+
+sub AUTOLOAD {
+  my $time = localtime;
```

Comment:
Might have been a good idea to set the time when you create the object. Otherwise you might find that your time is a little off when you actually come to call the methods